website/content/api-docs/secret/pki.mdx

```diff
@@ -3359,6 +3360,94 @@ $ curl \
 }
 ```
 
+### Sign Revocation List
+
+This endpoint allows generating a CRL based on the provided parameter data from any external
+source and signed by the specified issuer. Values are taken verbatim from the parameters provided.
+
+**This is a potentially dangerous endpoint and only highly trusted users should have access.**
```

Comment:
Do we usually use bold for these? Just curious :)

Reply:
Yup pretty much copy/paste from Sign Verbatim